orchagent/muxorch.cpp

```diff
@@ -1862,6 +1869,30 @@ bool MuxOrch::isStandaloneTunnelRouteInstalled(const IpAddress& neighborIp)
     return standalone_tunnel_neighbors_.find(neighborIp) != standalone_tunnel_neighbors_.end();
 }
 
+void MuxOrch::updateCachedNeighbors()
+{
+    if (!enable_cache_neigh_updates_)
+    {
+        SWSS_LOG_NOTICE("Skip process cached neighbor updates");
```

Comment:
lets change to INFO. this would be logged frequently otherwise

Reply:
`updateCachedNeighbors` is only called once in `OrchDaemon::warmRestoreAndSyncUp`, and it will only be called during warm reboot.